src/snowflake/snowpark/dataframe.py

```diff
@@ -766,6 +769,7 @@ def to_pandas(
         ...  # pragma: no cover
 
     @df_collect_api_telemetry
+    @open_telemetry(name="Dataframe.to_pandas")
```

Comment:
In my opinion: no need to add `Dataframe.` since the scope is `"snowflake.snowpark.dataframe"`.